taskcluster/ci/build/kind.yml

```diff
@@ -112,25 +101,12 @@ jobs:
         include-shippable-secrets: true
         run:
             geckoview-engine: geckoNightly
-            gradle-build-type: fennecNightly
+            gradle-build-type: fenixProduction
         treeherder:
             symbol: nightlySim(B)
 
-    beta:
-        attributes:
-            release-type: beta
-        include-release-version: true
-        include-shippable-secrets: true
-        filter-incomplete-translations: true
-        run:
-            geckoview-engine: geckoBeta
-            gradle-build-type: fenixBeta
-        run-on-tasks-for: [github-release]
-        treeherder:
-            symbol: beta(B)
-
     # XXX `production` is now the new nightly. We keep this name around while we officially remove
-    # `nightly` and `fennec-nightly`
+    # `nightly` and `fennec-nightly` off the Play Store
     production:
```

Comment:
I wonder if at some point we can rename these, because I could see this being a little confusing for people if they are getting into this app for the first time (but maybe that can happen as part of sebastian's cleaning up of the build flavors).

Reply:
I agree we should rename this especially because some mozilla-central jobs are looking up `nightly`. See https://github.com/mozilla-mobile/fenix/pull/12187#discussion_r449425823 for more details.